tests/testthat/_snaps/listing_lbl02a_rls.md

```diff
@@ -11,7 +11,7 @@
                                                                        Study   Last Dose of              from       Reference     Reference     Change     Abnormality
       Treatment          Lab Test (Unit)           Center/Patient ID    Day     Study Drug    Result   Baseline       Range         Range     Dec./Inc.       Flag    
       ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
-       Placebo    Alanine Aminotransferase (U/L)       701/1015         -7          0          27.0        0.0     10.0 -  29.0    6 - 34     40% / 40%               
+       Placebo    Alanine Aminotransferase (U/L)       701/1015         -7          0          27.0         NA     10.0 -  29.0    6 - 34     40% / 40%               
```

Comment:
hi @khatril , this is Baseline value, should we consider display this as 
1.  na_str = "<No data>"
2.  na_str = " "
3.  na_str = "-"

Reply:
it is the baseline so I think it best "-"